src/pages/index.js

```diff
@@ -219,7 +219,6 @@ function handleAddCardSubmit(evt) {
       .then((data) => {
         const cardEl = getCardElement(data);
         cardList.prepend(cardEl);
-        evt.target.reset();
         disableButton(cardSubmitBtn, validationConfig);
         closeModal(cardModal);
```

Comment:
`closeModal(cardModal);` is not needed here because it's already inside `handleSubmit`